docs/CHANGELOG.rst

```diff
@@ -22,6 +22,7 @@ Changed
 - Support Salmon reports in MultiQC and expose ``dirs_depth`` parameter
 - Expose transcript-level expression file in the ``salmon-quant``
   process
+- Give process ``upload-bedpe`` access to network
```

Comment:
This can be considered a bug fix, so please move under Fixed section. Also, bump the process version at the last digit, to `1.0.2`. Otherwise looks good.